qemu/tests/dump_guest_core.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Test dump-guest-core, this case will:
+
+    1) Start VM with dump-guest-core option.
+    2) Check host env.
+    3) Trigger a core dump in host.
+    4) Use gdb to check core dump file.
+    5) If dump-guest-core=on, use crash to check vmcore file
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    def check_env():
+        """
+        Check if host kernel version is same with guest
+        """
+        guest_kernel_version = session.cmd("uname -r").strip()
+        if host_kernel_version != guest_kernel_version:
+            test.cancel("Please update your host and guest kernel "
```

Comment:
Can you provide the information about guest's kernel and host's kernel in this message ?

Reply:
> > And the title of the patch also need to updated, for example,
> > spapr_vty_multi_backends: New case to test guest with multi backends of pci-serial
> 
> Those info seems should described in commit.

Hi @YongxueHong , I remembered that we need to have the tile of the patch like that, what's your opinions ? Thanks.

Reply:
> > > And the title of the patch also need to updated, for example,
> > > spapr_vty_multi_backends: New case to test guest with multi backends of pci-serial
> > 
> > 
> > Those info seems should described in commit.
> 
> Hi @YongxueHong , I remembered that we need to have the tile of the patch like that, what's your opinions ? Thanks.

Hi @MiriamDeng 
yes, like this PR is to new a case `dump_guest_core`, so I suggest the commit header like:
`dump_guest_core: xxxxxxxxx`

So for the original commit header `dump_guest_core.py:qemu dump new cases development`, @leidwang , you might not write the filename, just use the `dump_guest_core` to replace `dump_guest_core.py`.
Thanks.

Reply:
Ok, will update it,Thanks!